api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/PaymentCycleVoucherEntitlementTestDataFactory.java

```diff
@@ -46,7 +46,7 @@ public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlementMatc
 
     public static PaymentCycleVoucherEntitlement aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(
             LocalDate startDate, List<LocalDate> childrensDob) {
-        LocalDate birthdate = childrensDob.stream().min(LocalDate::compareTo).get();
+        LocalDate birthdate = childrensDob.stream().max(LocalDate::compareTo).get();
```

Comment:
Maybe this should be in its own method for clarity as it's obvious this can easily confuse.

Reply:
Updated